Jail/Assets/Assets/Scripts/Player.cs

```diff
@@ -368,6 +374,15 @@ void UpdateState()
                     UpdateConnectionState();
                 }
             }
+
+            if (inCheckpoint)
```

Comment:
(?) This code-block should be commented (like `//  use checkpoint when not in action`)